### PR TITLE
Move on stop server

### PR DESCRIPTION
**Pull request by uweenukr**

### CODE REVIEW

File: Assets/Mirror/Runtime/NetworkServer.cs [hunk 1]
```diff
@@ -33,6 +33,7 @@ public class NetworkServer : MonoBehaviour
 
         public NetworkConnectionEvent Connected = new NetworkConnectionEvent();
         public NetworkConnectionEvent Authenticated = new NetworkConnectionEvent();
+        public UnityEvent OnStopServer = new UnityEvent();
```
Comment (paulpach) [inline Assets/Mirror/Runtime/NetworkServer.cs]: rename it to "Stopped" to be consistent with the other events

Comment (uweenukr) [reply Assets/Mirror/Runtime/NetworkServer.cs]: fixed

File: Assets/Mirror/Runtime/NetworkServer.cs [hunk 2]
```diff
@@ -95,6 +96,8 @@ public void Shutdown()
         {
             if (initialized)
             {
+                Stopped.Invoke();
```
Comment (paulpach) [inline Assets/Mirror/Runtime/NetworkServer.cs]: We probably want to disconnect the clients first no?

Comment (uweenukr) [reply Assets/Mirror/Runtime/NetworkServer.cs]: fixed

**Pull request merged**